Formula/t/tinyxml.rb

```diff
@@ -23,8 +23,12 @@ class Tinyxml < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "ade5525899de7063ade79d1b0dec70ceef3d0acc08e1dc1b55e937cb539ad38d"
   end
 
+  deprecate! date: "2023-01-05", because: :deprecated_upstream
+
   depends_on "cmake" => :build
 
+  # sourceforge mentions to use tinyxml2
+
```

Comment:
why adding this comment?

Reply:
this is from a cherry-pick of https://github.com/Homebrew/homebrew-core/pull/119829/commits/3f57330df6aa4c670e5be28dbf9726aa0f33190a from https://github.com/Homebrew/homebrew-core/pull/119829, originally by @SMillerDev. I think the comment means to suggest what people can use instead of this deprecated formula. It could perhaps be moved to the line above the `deprecate!` statement? Let me know if I should make that change

Reply:
Sounds good

Reply:
works for me, thanks for the explanations. 👍

Reply:
comment moved and reworded in https://github.com/Homebrew/homebrew-core/pull/199811/commits/ea40f93931d275c347f2b3884f6c611f5a8f3ab5